### PR TITLE
[Azure Stack] Updates to Azs.Compute.Admin spec for autorest config

### PR DESCRIPTION
**Pull request by viananth**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/azsadmin/resource-manager/compute/readme.md [hunk 1]
```diff
@@ -56,32 +56,16 @@ csharp:
   clear-output-folder: true
 ```
 
-## Multi-API/Profile support for AutoRest v3 generators 
+# AutoRest v3 generators
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/compute/readme.md:59]: we dont need to change this part i think

File: specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json [hunk 2]
```diff
@@ -44,6 +44,12 @@
           {
             "$ref": "Disks.json#/parameters/SourceSharePathParameter"
           },
+          {
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json:47]: @zyxyoshine Havent you made this change already to master? you made this only in stackadmin?

**Review by bganapa — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]
LGTM

File: specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json (hunk 2 above)
Comment (zyxyoshine) [reply specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json:47]: @bganapa I believe they are in master. PR https://github.com/Azure/azure-rest-api-specs/pull/8747 has been merged.

**Comment by viananth** [comment]
@zyxyoshine could you please approve this PR if it looks good to you? Thanks!

**Review by zyxyoshine — APPROVED** [review]

**Review by zyxyoshine — APPROVED** [review]

**Comment by viananth** [comment]
@fengzhou-msft could you pls merge this PR if it looks good to you? Thanks!

**Comment by viananth** [comment]
@KrisBash could you pls merge this PR if it looks good? Thank you! @bganapa FYI

**Comment by bganapa** [comment]
@KrisBash Do you need anything to be fixed here? Could you pls merge this

**Comment by viananth** [comment]
@fengzhou-msft is there any changes you are expecting? Could you please merge if it looks good? Kris had already signed off.

**Comment by fengzhou-msft** [comment]
@viananth sorry for the delay, I do not have permission to force merge the PR. @akning-ms can you help merge this PR?

**Pull request merged**